index.js

```diff
@@ -1282,7 +1282,7 @@ var lighterhtml = (function (document,exports) {
   }; // generic content render
 
   function render(node) {
-    var value = update.apply(this, arguments);
+    var value = update.apply(this, Array.prototype.slice.call(arguments));
```

Comment:
this is historically a perf killer:
https://www.jstips.co/en/javascript/avoid-modifying-or-passing-arguments-into-other-functions-it-kills-optimization/
https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#3-managing-arguments

I hope you starts getting why I'm not super keen to play around with arguments here

Reply:
Oh, I was unaware of that
Thanks!